Manito/Manito/Screens/Main/UIComponent/CommonMissonView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CommonMissonView.swift
+//  Manito
+//
+//  Created by COBY_PRO on 2022/06/12.
+//
+
+import UIKit
+import SnapKit
+
+final class CommonMissonView: UIView {
+    
+    let missonName = "별별인사하기"
+    
+    // MARK: - property
+    
+    private lazy var title: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 공통미션"
+        label.textColor = .grey003
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    
+    private lazy var mission: UILabel = {
+        let label = UILabel()
+        label.text = missonName
+        label.numberOfLines = 2
+        label.textAlignment = .center
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 25)
+        return label
+    }()
+
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - func
+    
+    private func render() {
+        self.addSubview(title)
+        title.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.centerX.equalToSuperview()
```

Comment:
p1;

```suggestion
            $0.top.centerX.equalToSuperview()
```

이렇게 정렬해주세요~!

Reply:
중복되는걸 이렇게 하는 꿀팁... 배워갑니다.